src/js/views/globals/search/search-by-dob_views.js

```diff
@@ -0,0 +1,196 @@
+import { noop, find, get, map, debounce, each, extend, pick } from 'underscore';
+import dayjs from 'dayjs';
+import hbs from 'handlebars-inline-precompile';
+import { View, CollectionView } from 'marionette';
+
+import hasAllText from 'js/utils/formatting/has-all-text';
+
+import PicklistComponent from 'js/components/picklist';
+
+import InputWatcherBehavior from 'js/behaviors/input-watcher';
+import PicklistBehavior from 'js/behaviors/picklist-transport';
+
+const CLASS_OPTIONS = [
+  'childView',
+  'childViewEventPrefix',
+  'className',
+  'lists',
+];
+
+const CLASS_OPTIONS_ITEM = [
+  'itemClassName',
+  'itemTemplate',
+  'itemTemplateContext',
+];
+
+const Picklist = CollectionView.extend({
+  className: 'picklist__group',
+  tagName: 'li',
+  template: hbs`<ul></ul>`,
+  serializeCollection: noop,
+  childViewContainer: 'ul',
+  childViewEventPrefix: 'item',
+  modelEvents: {
+    'change:query': 'filter',
+  },
+  viewFilter(view) {
+    view.render();
+    const query = this.model.get('query');
+    return !query || !view.searchText || hasAllText(view.searchText, query);
+  },
+  initialize(options) {
+    this.mergeOptions(options, CLASS_OPTIONS_ITEM);
+  },
+  childViewOptions() {
+    const opts = pick(this, ...CLASS_OPTIONS_ITEM);
+    return extend({ state: this.model }, opts);
+  },
+});
+
+const PicklistsCollectionView = CollectionView.extend({
+  behaviors: [
+    InputWatcherBehavior,
+    PicklistBehavior,
+  ],
+  template: hbs`
+    <div class="modal__header patient-search__header">
+      <span class="modal__header-icon">{{far "magnifying-glass"}}</span>
+      <div class="js-clear-search-type button-primary patient-search__clear-type-button">
+        <span class="patient-search__clear-type-button-label">Date of Birth</span>{{far "xmark"}}
+      </div>
+      <input
+        type="date"
+        class="js-input patient-search__input"
+        placeholder="{{ @intl.globals.search.patientSearchViews.patientSearchPicklist.searchByDOB.placeholderText }}"
```

Comment:
Tried to set a placeholder of `MM/DD/YYYY` here. But as I learned, I guess you can't change the placeholder used for a `type="date"` input.

The default is `mm/dd/yyyy`:

<img width="540" alt="Screen Shot 2022-09-01 at 12 07 27 AM" src="https://user-images.githubusercontent.com/35355575/187835718-e8cadb4a-47a5-4929-9752-5c0a00caa48a.png">

Reply:
we're doing something for a placeholder for input date on the outreach login